VimeoNetworking/Sources/AuthenticationController.swift

```diff
@@ -564,9 +571,7 @@ final public class AuthenticationController
         {
             try self.setClientAccount(with: account, shouldClearCache: true)
             
-            let accountType: AccountStore.AccountType = (account.user != nil) ? .User : .ClientCredentials
-            
-            try self.accountStore.saveAccount(account, type: accountType)
+            try self.saveAccount(account)
```

Comment:
@ghking Don't think I have access to create a ticket on the AF board. Mind deleting this duplicate I created https://vimean.atlassian.net/browse/VIM-4979 and migrating this one https://vimean.atlassian.net/browse/VIM-4978 to Apple Frameworks?